src/pages/journal/journal.ts

```diff
@@ -20,15 +22,20 @@ export class JournalPage {
   constructor(
     public navCtrl: NavController,
     public navParams: NavParams,
-    private journalProvider: JournalProvider
+    private journalProvider: JournalProvider,
+    private faultsService: FaultsScorecardProvider,
+    private faultStore: FaultStoreProvider
   ) {}
 
   ionViewDidLoad() {
     this.journalProvider.getData('test@test.com').subscribe((data) => {
       this.journalSlots = data;
     });
+
+    this.faultsService.reset();
+    this.faultStore.reset();
```

Comment:
I don't think FaultsScorecardProvider is used anywhere, if it is it should be using the FaultStore